subnet/deployment-generator/docker/Dockerfile

```diff
@@ -1,20 +1,27 @@
 
 FROM golang:1.14 as builder
 
-#use node base image since it can already run the go binary
-# RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash - &&\  
-#     apt-get install -y nodejs
-# RUN apt-get update && apt-get install -y build-essential 
-# RUN npm install -g yarn
+ARG SUBNET_BRANCH=master    
 
 RUN git clone https://github.com/XinFinOrg/XDC-Subnet
 
-ARG SUBNET_BRANCH=master        
 #it is called 'BRANCH' but commits also work
 RUN cd XDC-Subnet && git checkout ${SUBNET_BRANCH} &&  make puppeth
 
+
+
+
+
 FROM node:18.15
```

Comment:
suggest to use node alpine image to minimize the image

Reply:
I tried that but it cannot execute the puppeth binary

Reply:
ok